pkg/sources/sources.go

```diff
@@ -26,6 +26,8 @@ type Chunk struct {
 	Data []byte
 	// Verify specifies whether any secrets in the Chunk should be verified.
 	Verify bool
+	// Keywords found within the chunk's Data.
+	Keywords map[string]bool
```

Comment:
Why did you add this to the Chunk? Are you thinking that we might want to analyze which keywords get hit?

Reply:
@dustin-decker whoops! When I was grokking the code at first I thought we would need `Keywords` as part of `Chunk` for the lookup but now I see we don't. Nice catch. I'll update this tomorrow when I'm on my laptop